### PR TITLE
Merge with manubot-rootstock on 2017-09-28

### PR DESCRIPTION
**Pull request by dhimmel**
Up to https://github.com/greenelab/manubot-rootstock/commit/72189b5bdff80573e0d89f7c2d30975753a57b2e

Here's the resulting PDF: [manuscript.pdf](https://github.com/greenelab/meta-review/files/1342117/manuscript.pdf).

Notice the odd margins. Perhaps it's because of the really long inline code.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,13 +5,12 @@ branches:
   only:
     - master
 before_install:
-  - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
+  - wget https://repo.continuum.io/miniconda/Miniconda3-4.3.21-Linux-x86_64.sh
```
Comment (agitter) [inline .travis.yml:8]: I see the build is failing when we use the latest Miniconda.  I think we should be safe to merge and then confirm the merged version runs on Travis CI.

File: README.md [hunk 2]
```diff
@@ -1,30 +1,47 @@
...
 
+<!-- usage note: edit the H1 title above to personalize the manuscript -->
+
+[![HTML Manuscript](https://img.shields.io/badge/manuscript-HTML-blue.svg)](https://greenelab.github.io/meta-review/)
+[![PDF Manuscript](https://img.shields.io/badge/manuscript-PDF-blue.svg)](https://greenelab.github.io/meta-review/manuscript.pdf)
+[![Build Status](https://travis-ci.org/greenelab/manubot-rootstock.svg?branch=master)](https://travis-ci.org/greenelab/meta-review)
+
+## Manuscript description
+
+<!-- usage note: edit this section.
```
Comment (agitter) [inline README.md]: I think we can remove this comment.

File: README.md [hunk 3]
```diff
@@ -1,30 +1,47 @@
 # Deep Review Collaborative Workflow and Manubot System
 
+<!-- usage note: edit the H1 title above to personalize the manuscript -->
+
+[![HTML Manuscript](https://img.shields.io/badge/manuscript-HTML-blue.svg)](https://greenelab.github.io/meta-review/)
+[![PDF Manuscript](https://img.shields.io/badge/manuscript-PDF-blue.svg)](https://greenelab.github.io/meta-review/manuscript.pdf)
+[![Build Status](https://travis-ci.org/greenelab/manubot-rootstock.svg?branch=master)](https://travis-ci.org/greenelab/meta-review)
```
Comment (agitter) [inline README.md]: Need the meta-review Travis CI svg.

File: README.md [hunk 4]
```diff
@@ -1,30 +1,47 @@
...
+Actual manuscript instances will clone this repository (see [`SETUP.md`](SETUP.md)) and replace this paragraph with a description of their manuscript.
+-->
+
 This manuscript presents the benefits of writing collaborative reviews in the open and the Manubot System for automating large portions of the build process.
 The results are derived from the authors' experience with the collaborative [Deep Review](https://github.com/greenelab/deep-review).
 [Feedback](https://github.com/greenelab/meta-review/issues) and minor contributions (e.g. typo corrections) are welcome.
 Major contributions are not being solicited at this time.
 
-The most current version of the `master` branch is built by continuous integration and [available via gh-pages](https://greenelab.github.io/meta-review/).
-To see what's incoming, check the [open pull requests](https://github.com/greenelab/meta-review/pulls).
```
Comment (agitter) [inline README.md:9]: Should we keep the link about pull requests?

File: SETUP.md [hunk 5]
```diff
@@ -0,0 +1,179 @@
+# Cloning manubot-rootstock to create a new manuscript
```
Comment (agitter) [inline SETUP.md]: Can we remove this file?

**Review by agitter — APPROVED** [review]
A few comments and small suggestions, but LGTM in general.

**Comment by agitter** [comment]
I'm not worried about the margins right now.  We can check again after removing those long TODOs.

File: SETUP.md (hunk 5 above)
Comment (dhimmel) [reply SETUP.md]: Sure

**Pull request merged**